clients/src/main/java/org/apache/kafka/clients/consumer/internals/ClassicKafkaConsumer.java

```diff
@@ -107,7 +107,7 @@
  * group protocol (from KIP-848) introduces allows users continue using the legacy "classic" group protocol.
```

Comment:
nit: introduces allow...not introduced here but makes sense to review the javadoc after the rename

Reply:
Updated. Thanks!